src/OpenTelemetry.Instrumentation.AspNet/Implementation/RequestDataHelper.cs

```diff
@@ -40,10 +43,37 @@ public RequestMethodHelper()
             };
     }
 
+    public static string GetHttpProtocolVersion(HttpRequest request)
+    {
+        return GetHttpProtocolVersion(request.ServerVariables["SERVER_PROTOCOL"]);
+    }
+
+    public void SetHttpMethodTag(Activity activity, string originalHttpMethod)
+    {
+        var normalizedHttpMethod = this.GetNormalizedHttpMethod(originalHttpMethod);
```

Comment:
nit: 
```csharp

if (this.knownMethods.TryGetValue(method, out var normalizedMethod))
{
   activity.SetTag(SemanticConventions.AttributeHttpRequestMethod, normalizedMethod);
}
else
{
    activity.SetTag(SemanticConventions.AttributeHttpRequestMethod, OtherHttpMethod);
    activity.SetTag(SemanticConventions.AttributeHttpRequestMethodOriginal, method);
}


```

Reply:
It does not means the same. Lets imagine that we have `GeT` method.

It should set `http.request.method` to `GET` abd `http.request.method_original` to `GeT`. Your proposal change it to just `http.request.method` to `GET`.

Reply:
Ah got it. This is where its coming from

> HTTP method names are case-sensitive and http.request.method attribute value MUST match a known HTTP method name exactly. Instrumentations for specific web frameworks that consider HTTP methods to be case insensitive, SHOULD populate a canonical equivalent. Tracing instrumentations that do so, MUST also set http.request.method_original to the original value.

Another candidate for https://github.com/open-telemetry/opentelemetry-dotnet/issues/5461

Reply:
will do